packages/forms-web-app/src/views/submit-appeal/application-saved.njk

```diff
@@ -3,16 +3,6 @@
 {% set title = "Your appeal for application " + applicationNumber + " has been saved - Appeal a planning decision - GOV.UK" %}
 {% block pageTitle %}{{ title }}{% endblock %}
 
-{% block backButton %}
-  {{ govukBackLink({
-    text: 'Back',
-    href: '/full-appeal/submit-appeal/task-list',
-    attributes: {
-      'data-cy': 'back'
-    }
-  }) }}
-{% endblock %}
-
```

Comment:
Assuming that no back button is required, correct?

Reply:
this template uses the default 'last page you visited' back button, which is the required behaviour for the ticket - I have removed the hardcoded task-list back button